src/copied/_locales/ru/extension.ru.json

```diff
@@ -684,6 +684,7 @@
     "reissue": "Перевыпустить",
     "transfer": "Перевести",
     "txid": "ID транзакции",
+    "hash": "Hash",
```

Comment:
@msmolyakov do we need to translate this into all languages?

Reply:
Yes, I'm waiting for the translations and will provide them later